pandas/core/series.py

```diff
@@ -450,7 +452,9 @@ def get_values(self):
 
     @property
     def asobject(self):
-        """DEPRECATED: Use ``astype(object)`` instead.
+        """
+        .. deprecated :: 0.21.1
+            Use ``astype(object) instead.
```

Comment:
I am not 100% sure, but might need a blank line after these directives (you can build and see if they look ok)